thetis/options.py

```diff
@@ -361,9 +361,29 @@ class TidalTurbineFarmOptions(FrozenHasTraits, TraitType):
         0.0, help='Average power production per turbine required to break even')
 
 
+class NonhydrostaticModelOptions(FrozenHasTraits):
+    """Options for non-hydrostatic models"""
+    name = 'Non-hydrostatic 2D/3D models'
+    solve_nonhydrostatic_pressure = Bool(False, help='Solve equations with the non-hydrostatic pressure').tag(config=True)
+    # use_3d_ale_nh = Bool(False, help='Use a 3D ALE non-hydrostatic model accounting for vertical mesh movement').tag(config=True)
+    # use_3d_sigma_nh = Bool(False, help='Use a 3D sigma-coordinate non-hydrostatic model in a fixed domain').tag(config=True)
+    update_free_surface = Bool(True, help='Update free surface elevetion').tag(config=True)
```

Comment:
elev**a**tion. Also be a bit more specific the help: "Update free surface elevation after pressure projection/correction step"(whichever you think is more correct)